src/lang/ru-RU.json

```diff
@@ -110,8 +109,8 @@
   "barracks_value_512": "Средний дальнего боя сил Света",
   "barracks_value_1024": "Верхний ближнего боя сил Света",
   "barracks_value_2048": "Верхний дальнего боя сил Света",
-  "benchmarks_description": "{0} {1} равен или больше {2}% недавних показателей на этом герое",
-  "fantasy_description": "{0} на {1} балла",
+  "benchmarks_description": "https://www.opendota.com/players/121943326",
```

Comment:
It's real correct?

Reply:
Nope, someone has submitted bad translations for these. If you could provide a correct translation please do so!

Reply:
@howardchung where I can do that?